app/model/cvegroup.py

```diff
@@ -4,7 +4,7 @@
 
 
 pkgname_regex = r'^([a-z\d@\.\_\+-]+)$'
-pkgver_regex = r'^(\d+:)?([\w]+[\._+]*)+\-\d+$'
+pkgver_regex = r'^(\d+:)?([\w]+[\._+]*)+\-\d[\d\.]*$'
```

Comment:
@rgacogne this regex has backreferences and results in high resource consumption with an evil input